torch/csrc/generic/methods/TensorMath.cwrap

```diff
@@ -1891,3 +1891,33 @@ static const char *R = &__R;
       if_false: N
       default: N
 ]]
+
+[[
+  name: btrifact
+  cname: btrifact
+  defined_if: defined(TH_REAL_IS_FLOAT) || defined(TH_REAL_IS_DOUBLE) || CUDA_FLOAT || CUDA_HALF || CUDA_DOUBLE
+  with_stateless: true
+  return: argument 0,1,2
+  arguments:
+    - arg: THTensor* result
+      output: True
+    - arg: THIntegerTensor* pivots
+      output: True
+    - arg: THIntegerTensor* info
+      output: True
+    - THTensor* self
+]]
+
+[[
+  name: btrisolve
+  cname: btrisolve
+  defined_if: defined(TH_REAL_IS_FLOAT) || defined(TH_REAL_IS_DOUBLE) || CUDA_FLOAT || CUDA_HALF || CUDA_DOUBLE
+  with_stateless: true
+  return: argument 0
+  arguments:
+    - arg: THTensor* result
+      output: True
+    - THTensor* Atf
```

Comment:
Sure, I've clarified this name to `A_LU_data` and the pivots to `A_LU_pivots` 69454409e1b0048f5b179d4a124b70836cc4cacd